src/melee/ft/chara/ftNess/ftNess_AttackHi4.c

```diff
@@ -383,21 +368,26 @@ void ftNess_YoyoSetChargeDamage(
     s32 filler;
 
     fp = getFighterPlus(fighter_gobj);
-    fp->nessVars[0].AttackHi4.yoyoCurrentFrame =
-        14; // Current animation frame, integer //
+
+    // Current animation frame, integer
+    fp->nessVars[0].AttackHi4.yoyoCurrentFrame = 14;
+
     yoyo_GObj = fp->sa.ness.x222C_yoyoGObj;
-    if (yoyo_GObj != NULL) {
+
+    if (yoyo_GObj) {
```

Comment:
I'm so used to looking at C# code that I normally lean toward `!= NULL`, but based on asserts I think HAL always does it this way. What do you think we should use?

Reply:
I also like the explicit NULL comparison and so does Psi based on the style guide. It can't be used in asserts but otherwise I don't think it ever changes codegen. So I'd say keep it.

Reply:
Okay, will revert.